src/main/java/net/mcreator/ui/modgui/CustomGUIGUI.java

```diff
@@ -36,6 +36,7 @@
 public class CustomGUIGUI extends ModElementGUI<GUI> {
 
 	private WYSIWYGEditor editor;
+	public static WYSIWYGEditor staticComponentList = new WYSIWYGEditor(null, true);
```

Comment:
This will not work, we do not and can not have static references in object-driven app